spec/models/course_enrichment_spec.rb

```diff
@@ -215,7 +226,6 @@
 
     context "fee based course" do
       it { is_expected.to validate_presence_of(:fee_uk_eu).on(:publish) }
-      it { is_expected.to validate_presence_of(:required_qualifications).on(:publish) }
```

Comment:
As far as I can see the validation doesn't depend on the course being fee/salary-based so just removing this so that it doesn't break in 2022. Does that make sense?